tutorials/3d/portals/fundamentals.rst

```diff
@@ -0,0 +1,102 @@
+3. By manually editing the points that define the convex hull, in the room inspector.
+
+While the first option can be all that is required, particularly with simple rooms, or for pre-production, the power of the manual bounds gives you ultimate control (at the expense of a small amount of editing). You can also combine the two approaches, perhaps using automatic bounds for most rooms but manually editing problem areas.
+
+The automatic method is used whenever a manual bound is not supplied.
+
+*A simple pair of rooms. The portal margin is shown with translucent red, and the room hulls are shown with green wireframe.*
+
+.. image:: img/simple_room.png
+
+Portals
+~~~~~~~
+If you create some rooms, place objects within them, then convert the level in the editor, you will see the objects in the rooms appearing and showing as you move between rooms. There is one problem however! Although you can see the objects within the room that the camera is in, you can't see to any neighbouring rooms! For that we need portals.
+
+:ref:`Portal<class_Portal>`\ s are special convex polygons. You position over the openings between rooms, in order to allow the system to see between them. You can create a portal node directly in the editor. The default portal has 4 points and behaves much like a ``plane`` :ref:`MeshInstance<class_MeshInstance>`. You can add or remove points using the inspector. A portal will require at least 3 points to work (in order to form a polygon rather than a point or line).
+
+Portals only need to be placed in one of each pair of neighbouring rooms (the *'source room'*) - the system will automatically make them two way unless you choose otherwise in the portal settings. The portal normal should face *outward* from the source room. The front face should be visible from *outside* the room. The editor gizmo indicates the direction the portal is facing with an arrow, and a different color for each face.
+
+.. image:: img/portal_inspector.png
+
+Like rooms, portals also follow a naming convention which is as follows:
+- Prefix ``Portal_``
+- Optional : You can add a suffix of the room that the portal will lead to ('destination room'). E.g. ``Portal_kitchen``
+
+The suffix is optional - in many cases the system can automatically detect the nearest room that you intended to link to and do this for you. It is usually only in problem areas you will need to use the suffix.
+
+In rare cases you may end up with two or more portals that you want to give the same name, because they lead into the same destination room. But Godot does not allow duplicate names! The solution to this is the wildcard character `*`. If you place a wildcard at the end of the name, the rest of the characters will be ignored. E.g. ``Portal_Kitchen*1``, ``Portal_Kitchen*2``.
```

Comment:
```suggestion
In rare cases, you may end up with two or more portals that you want to give the same name, because they lead into the same destination room. However, Godot does not allow duplicate node names at the same level in the scene tree. The solution to this is the wildcard character ``*``. If you place a wildcard at the end of the name, the rest of the characters will be ignored. For example, name your nodes ``Portal_Kitchen*1`` and ``Portal_Kitchen*2``.
```